README.md

```diff
@@ -1,2 +1,136 @@
-# consume-more-stuff
-A custom CMS for personal product sale
+# Consume More Stuff
+> A custom CMS for personal product sale
+
+## Introduction
+Build a Content Management System using:
+
+- __Backend__
+    - **Express** as the application server
+    - **Postgres** as the database server
+    - **Sequelize.js** as the ORM
+    - **passport.js** for Authentication and Authorization
+
+- __Frontend__
+	- **React.js** or **AngularJS** as FE framework
+	- **Sass** for styling
+	- **Webpack** for bundling	
+- __Stretch Add-ons__
+	
+	Some stretch and polish goals will use additional technologies.
+	
+	- **AWS S3** for storing product images
+	- **mailgun** or **sendgrid** for sending email
+	
+Your team will be building a minimal CMS to allow users to list personal items for sale.
+The application will allow unauthorized users to browse the listed products. To post an Item for sale, a User account will be required. The backend should provide Restful api endpoints. The frontend should be a SPA that provides a smooth user experience.
+
+You will need to work as a team to complete this project. To facilitate team communication you will be using [Trello](https://trello.com/) to track tasks and stay on track.
+
+
+## Business Requirements
+
+### MVP
+
+#### Unauthenticated user
+- Home page should list top viewed `Items` by `Category`.
+- Home page should provide a way to view **all** `Items`.
+- Home page should provide a view of `Items` by `Category`.
+- Home page should show a subset of key details of the `Items`, along with one image.
+- Home page should have a login button.
+- `Item` detail page should contain:
+	- description (required)
+	- price
+	- manufacturer or make
+	- model name or number
+	- condition (required)
+	- category (required)
+	- dimensions
+	- when the `Item` was initially published
+	- when the `Item` was last updated
+	- notes / details
+
+#### Authenticated User
+- Should be able to view the same home page and details that an unauthenticated user can.
+- `User` home page should contain: 
+	- A way to add new `Items`.
+	- A way to logout.
+	- display the current `Users` name
+	- List of `Items` currently published for sale.
+	- List of `Items` pending publishing.
```

Comment:
Should some states of items be considered stretch goals? I think it's relevant, but it could take time to work through. Maybe start with just published and mark as sold? Binary state of what should show in public view.